### PR TITLE
[next][build-utils] Add support for cron to vercel.json

### PR DESCRIPTION
**Pull request by AndyBitz**
Extends `vercel.json` to support setting `cron` on `functions`.

Supports Serverless Functions and Edge Functions.

### CODE REVIEW

File: packages/build-utils/src/schemas.ts [hunk 1]
```diff
@@ -31,6 +31,11 @@ export const functionsSchema = {
           type: 'string',
           maxLength: 256,
         },
+        cron: {
+          type: 'string',
+          minLength: 9,
```
Comment (Ethan-Arrowood) [inline packages/build-utils/src/schemas.ts:36]: Does this represent "* * * * *"? Is that the minimum possible cron string?

File: packages/next/test/unit/build.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+const path = require('node:path');
+const rimraf = require('rimraf');
```
Comment (Ethan-Arrowood) [inline packages/next/test/unit/build.test.js]: Don't we have fs-extra for this?

**Review by Ethan-Arrowood — COMMENTED** [review]
This is looking really good! Excited for CRON job support 😁

File: packages/build-utils/src/schemas.ts (hunk 1 above)
Comment (AndyBitz) [reply packages/build-utils/src/schemas.ts:36]: Exactly, 5 fields + 4 spaces is the minimum

File: packages/next/test/unit/build.test.js (hunk 2 above)
Comment (AndyBitz) [reply packages/next/test/unit/build.test.js]: Ah, good point, forgot about that 👀

**Review by Ethan-Arrowood — APPROVED** [review]

File: packages/next/test/unit/build.test.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+const path = require('node:path');
```
Comment (vvo) [inline packages/next/test/unit/build.test.js:1]: The name of the test is generic (= no cron specific), was that a missing test for next?

**Review by vvo — APPROVED** [review]

File: packages/next/test/unit/build.test.js (hunk 3 above)
Comment (AndyBitz) [reply packages/next/test/unit/build.test.js:1]: It is sort of a new way to test the next builder. The other unit tests, test some specific functions from the package, while the integration tests create actual Deployments on Vercel. This is sort of a middle ground, where it tests the main build function from the package, but without creating a Deployment.

**Pull request merged**